include/exadg/convection_diffusion/spatial_discretization/operator.cpp

```diff
@@ -826,11 +830,76 @@ template<int dim, typename Number>
 void
 Operator<dim, Number>::update_after_grid_motion()
 {
-  // update SIPG penalty parameter of diffusive operator which depends on the deformation
-  // of elements
   if(param.diffusive_problem())
   {
+    // update SIPG penalty parameter of diffusive operator which depends on the deformation
+    // of elements
     diffusive_kernel->calculate_penalty_parameter(*matrix_free, get_dof_index());
+
+    // update the diffusivity coefficients which depend on the quadrature point coordinates
+    // and may depend on the time, e.g. in case of an analytical coefficient function.
+    // However, the ALE interface does not support passing in the time here, so the coefficients
+    // are calculated with the simulation start time.
+    calculate_diffusivity(this->param.start_time);
```

Comment:
The problem is that this code is not placed correctly in `update_after_grid_motion()`. Also as a user, I can not expect that for a time-dependent problem on a non-moving mesh, the calculation of a time-dependent diffusivity will be done in a function that explicitly relates to grid-motion (ALE) problems.